__tests__/app.test.js

```diff
@@ -45,7 +45,40 @@ describe("GET /api/topics", () => {
           expect(topic).toHaveProperty('description');
         })
       });
-    });
+  });
+  test("404: Responds with Endpoint Not Found when the endpoint is incorrect/non-existent", () => {
```

Comment:
This should be just one test. This error is not related to any particular endpoint.